gateway/ln-gateway/proto/gateway_lnrpc.proto

```diff
@@ -245,6 +248,11 @@ message OpenChannelRequest {
   uint64 push_amount_sats = 3;
 }
 
+message CloseChannelsWithPeerRequest {
+  // The public key of the node we're closing channels with.
+  string pubkey = 1;
```

Comment:
Can we use bytes instead of string?

Reply:
Done. The other liquidity API calls already use a string instead of bytes, but I'll fix this in a separate PR.